common/beerocks/bcl/include/bcl/network/network_utils.h

```diff
@@ -116,6 +116,17 @@ class network_utils {
     static bool linux_iface_is_up(const std::string &iface);
     static bool linux_iface_is_up_and_running(const std::string &iface);
 
+    /**
+     * @brief Gets the speed of a network interface.
+     *
+     * @param[in] iface Name of the network interface.
+     * @param[out] speed On success, speed of the network interface as defined in SPEED_* macros
```

Comment:
Excellent that you document the unit here.

Maybe add that it is in Mbps - otherwise you need to look at ethtool.h to discover that :-)

(nitpick)

Reply:
Done